### PR TITLE
Minor: added seconds to human-readable format scale for test case graph

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes:

Minor: added seconds to human-readable format scale for test case graph

image

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Comment by TeddyCr** [comment]
@ShaileshParmar11 you need to treat it as second. The millisecond value is a bug that will be fixed (we have an open PR)

File: openmetadata-ui/src/main/resources/ui/src/components/Database/Profiler/TestSummary/TestSummaryGraph.tsx [hunk 1]
```diff
@@ -165,6 +174,14 @@ function TestSummaryGraph({
     setActiveMouseHoverKey('');
   };
 
+  // Todo: need to find better approach to create dynamic scale for graph, need to work with @TeddyCr for the same!
+  const formatYAxis = (value: number) => {
+    // table freshness will always have output value in milliseconds
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/Database/Profiler/TestSummary/TestSummaryGraph.tsx:179]:
```suggestion
    // table freshness will always have output value in seconds
```

File: openmetadata-ui/src/main/resources/ui/src/components/Database/Profiler/TestSummaryCustomTooltip/TestSummaryCustomTooltip.component.tsx [hunk 2]
```diff
@@ -78,7 +80,10 @@ const TestSummaryCustomTooltip = (
           {startCase(key)}
         </span>
         <span className="font-medium" data-testid={key}>
-          {value}
+          {/* freshness will always be in millisecond  */}
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/Database/Profiler/TestSummaryCustomTooltip/TestSummaryCustomTooltip.component.tsx:83]:
```suggestion
          {/* freshness will always be in second  */}
```

File: openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.test.ts [hunk 3]
```diff
@@ -138,6 +139,29 @@ describe('Tests for CommonUtils', () => {
       });
     });
 
+    // formatTimeFromMilliseconds test
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.test.ts:142]:
```suggestion
    // formatTimeFromSeconds test
```

File: openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.test.ts [hunk 4]
```diff
@@ -138,6 +139,29 @@ describe('Tests for CommonUtils', () => {
...
+    it('formatTimeFromMilliseconds formatter should format mills to human readable value', () => {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.test.ts:143]:
```suggestion
    it('formatTimeFromSeconds formatter should format mills to human readable value', () => {
```

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**